src/client/components/AddFriendModal/AddFriendModal.jsx

```diff
@@ -69,12 +69,15 @@ class AddFriendModal extends Component {
         window.location.reload();
       })
       .catch(function(error) {
-        if (error.response && error.response.status == 401) {
+        if (error.response && error.response.status === 401) {
```

Comment:
use `switch`

```js
.catch (function(error) {
  if (error.response) {
    switch (error.response.status) {
      case 401:
        instance.routeChange(PATHS.landingPage);
        return;
      case 404:
        alert(`Could not find user with email '${email}'.`);
      default:
        alert(error.message);
        console.log(error);
        return
    }
  }
  alert(error.message);
  console.log(error);
})
```

Reply:
but then the default will be repeated right? i think i prefer to use if else.